test-cargo-miri/test.stderr-proc-macro.ref

```diff
@@ -0,0 +1 @@
+Running unit tests of `proc-macro` crate is not currently supported by Miri.
```

Comment:
Doesn't this mean https://github.com/rust-lang/miri/blob/master/test-cargo-miri/subcrate/test.rs is now dead code? That's not good, the code in there is testing some subtle things around working directories.

It might be better to add a new proc-macro-crate to the test workspace, instead of repurposing the existing subcrate.

Reply:
Oh, wait, am I mixing up unit tests and `[[test]]` tests?

Reply:
It's not dead code. That output is at stdout:
https://github.com/rust-lang/miri/blob/b1a8a375175ad95303985acb6247c7098dcd775e/test-cargo-miri/test.subcrate.stdout.ref#L6
And still tested:
https://github.com/rust-lang/miri/blob/b1a8a375175ad95303985acb6247c7098dcd775e/test-cargo-miri/run-test.py#L105

Reply:
Yeah I mixed up unit tests with integration tests.